[TS-978] fixed attribute presence=optional (#4)
th2-net/th2-codec-fix-orchestra: README.md

## Changelog

### v0.2.1

#### Fixed:

> * Attribute `presence="optional"` (optional tags were not checked correctly)

Reviewer: It is not an original problem. The reason why the rule did not work is that the MessageScope always returns a field even if it does not exist. We should describe that in the change log

Reviewer: Please, correct the description for changelog

> * Non-existent fields (messageScope returns fields, even if the message do not contain these fields, instead of returning null)

Reviewer: ```suggestion
* Non-existent fields (`messageScope` returns a field, even if the message does not contain this field, instead of returning null)
```
